packages/auth/src/graphql/schema.graphql

```diff
@@ -95,3 +129,6 @@ enum GrantState {
   "grant was rejected"
   REJECTED
 }
+
+scalar UInt8
+scalar UInt64
```

Comment:
These scalars would need to be added to the resolvers if you would like to use them. See: `packages/backend/src/graphql/resolvers/index.ts`. Otherwise, you can leave adding limits for a separate PR (if necessary)